MountSionCBSSecondary/SchoolWebsite
# Aup policy 2

# Description

Added AUP html

## What

Added AUP html and modified policies.html

## Why

To add AUP Policy to website

## Verification Steps

- create http-server 
- test changes are present on local server
- verify policy loaded correctly

# Checklist:
